python/ray/tests/test_basic_5.py

```diff
@@ -223,6 +223,33 @@ def sys_path():
     subprocess.check_call(["python", "-m", "package.module2"])
 
 
+def test_worker_kv_calls(monkeypatch, shutdown_only):
+    monkeypatch.setenv("TEST_RAY_COLLECT_KV_FREQUENCY", "1")
+    ray.init()
+
+    @ray.remote
+    def get_kv_metrics():
+        from time import sleep
+
+        sleep(2)
+        return ray._private.gcs_utils._called_freq
+
+    freqs = ray.get(get_kv_metrics.remote())
+    # So far we have the following gets
+    """
+    b'fun' b'IsolatedExports:01000000:\x00\x00\x00\x00\x00\x00\x00\x01'
+    b'fun' b'FunctionsToRun:01000000:\x12\x9b\xea\xa39\x01...'
+    b'fun' b'IsolatedExports:01000000:\x00\x00\x00\x00\x00\x00\x00\x02'
+    b'cluster' b'CLUSTER_METADATA'
```

Comment:
this is also something we can avoid?

Reply:
cluster_metadata I'm not sure. But for the isolated exportes/ functions to run, once we support code path setup in runtime env, we'll be able to avoid this.